server/asset/common.go

```diff
@@ -26,6 +26,8 @@ type Backend interface {
 	// the blockchain immediately. The redeem script is required in order to
 	// calculate sigScript length and verify pubkeys.
 	Contract(coinID []byte, redeemScript []byte) (Contract, error)
+	// TxData fetches the same data returned by a Contract's TxData method.
```

Comment:
Looks like you missed https://github.com/decred/dcrdex/pull/804/files#r557809395
```suggestion
	// TxData fetches the raw transaction data of the specified coin.
```